RATapi/utils/enums.py

```diff
@@ -48,15 +46,19 @@ class Display(RATEnum):
     Final = "final"
 
 
-class Strategies(Enum):
-    """Defines the available options for strategies"""
+class Strategies(RATEnum):
+    """Defines the available options for differential evolution strategies"""
+
+    Random = "random"
+    LocalToBest = "local to best"
+    BestWithJitter = "best w/ jitter"
```

Comment:
I'm not a great fan of this one - I think it'd be difficult to remember the "w/"

Reply:
Would "best jitter" work?

Reply:
sure - i'm not too picky on what the strings are because anything is more descriptive than "3"